Tests/FluentDarkModeKitUITests/DarkModeKitUITests.swift

```diff
@@ -32,9 +32,21 @@ final class DarkModeKitUITests: XCTestCase {
     _test("UIPageControl")
   }
 
+  func testUILabel() {
+    _test("UILabel")
+  }
+
+  func testUIImageView() {
+    _test("UIImageView")
+  }
+
   func _test(_ className: String) {
```

Comment:
Perhaps we can refactor this in the future. Maybe change to something like
```swift
func test(_ viewType: UIView.Type) {
  let className = String(describing: viewType)
  ...
}

test(UILabel.self)
```

Reply:
sounds good! probably the underscore needs still to be there in order not to be picked up by tests.

Reply:
I still don't think we should use underscore prefix in Swift. Maybe we can find a better name without the `test` prefix.